README.md

```diff
@@ -44,6 +44,9 @@ The Elasticsearch behaviour and properties can be configured through the followi
 *   **plugins** [defaultValue=""]
     > the list of plugins to install in each Elasticsearch instance before starting it (see the [Plugins](#plugins) section for details)
 
+*   **instanceSettings** [defaultValue=""]
+    > the list of settings to apply to corresponding Elasticsearch instances before starting (eg. `<instanceSettings><properties><name>NodeName</name></properties></instanceSettings>`), if the list is larger than `instanceCount`, the extra items are ignored
```

Comment:
I would also note that the names of thse properties would be passed to the ES instance(s) via the `-E` syntax, and link to https://www.elastic.co/guide/en/elasticsearch/reference/6.1/windows.html#msi-installer-command-line-configuration (it's the windows installation page, but it's the only page in the ES doc which seem to describe the -E usage in a bit more details).
I would also note that the list can be shorter than `instanceCount`, in which case no extra config will be applied to the remaining instances.